src/Uno.UWP/UI/Notifications/ScheduledToastNotification.cs

```diff
@@ -0,0 +1,51 @@
+using Uno.Logging;
+using Uno.Extensions;
+using Microsoft.Extensions.Logging;
+using Windows.Data.Xml.Dom;
+using System;
+
+namespace Windows.UI.Notifications
+{
+
+	public partial class ScheduledToastNotification
+	{
+		public ScheduledToastNotification(XmlDocument content, DateTimeOffset deliveryTime)
+		{
+			Content = content;
+			DeliveryTime = deliveryTime;
+		}
+
+		public XmlDocument Content { get; internal set; }
+		public DateTimeOffset DeliveryTime { get; internal set; }
+
+		private string _tag;
+
+		public string Tag

```

Comment:
Can this be null in UWP ? If it does, let's mark it as nullable.

Reply:
Cannot be null - added test in setter.